service/src/main/kotlin/fi/espoo/evaka/invoicing/service/VoucherValueDecisionGenerator.kt

```diff
@@ -109,7 +110,7 @@ private fun generateNewValueDecisions(
     families: List<FridgeFamily>,
     allPlacements: Map<ChildWithDateOfBirth, List<Pair<DateRange, PlacementWithServiceNeed>>>,
     prices: List<FeeThresholds>,
-    voucherValues: List<VoucherValue>,
+    voucherValues: Map<ServiceNeedOptionId, List<ServiceNeedOptionVoucherValue>>,
```

Comment:
Just noticed that the date ranges of `voucherValues` is missing from `val periods`, can you add them now? 😅

Reply:
Added 👍